src/domain/CheckNotificationStatusRepository.ts

```diff
@@ -15,20 +17,37 @@ export type CheckNotificationStatusRecord = {
   output: Response<200, NewNotificationRequestStatusResponse> | Response<403, UnauthorizedMessageBody> | Response<404>;
 };
 
-export const makeNewNotificationRequestStatusResponse = (
-  record: NewNotificationRecord
-): O.Option<NewNotificationRequestStatusResponse> => {
-  // eslint-disable-next-line sonarjs/no-small-switch
-  switch (record.output.statusCode) {
-    case 202:
-      return O.some({
-        ...record.output.returned,
-        ...record.input.body,
-        notificationRequestStatus: '__WAITING__',
-      });
-    default:
-      return O.none;
-  }
-};
+export const getNewNotificationRequestStatusResponse = (
```

Comment:
```suggestion
const getNewNotificationRequestStatusResponse = (
```
this method is only called on line 25